internal/pkg/agent/cmd/inspect.go

```diff
@@ -204,25 +212,11 @@ func printMapStringConfig(mapStr map[string]interface{}, streams *cli.IOStreams)
 }
 
 func printConfig(cfg *config.Config, l *logger.Logger, streams *cli.IOStreams) error {
```

Comment:
Looks like the logger is no longer being used by this function so it's parameter, `l`, can be removed.